query-engine/connectors/mongodb-query-connector/src/root_queries/write.rs

```diff
@@ -495,12 +497,22 @@ pub async fn query_raw<'conn>(
 
                 match operation {
                     MongoOperation::Find(filter, options) => {
-                        let cursor = coll.find_with_session(filter, options, session).await?;
+                        let unwrapped_filter = filter.clone().unwrap_or_default();
+                        let project = Document::default();
```

Comment:
Why are we always passing an empty projection instead of using the projection in the `FindOptions`?